src/replication.c

```diff
@@ -770,11 +770,14 @@ void syncCommand(client *c) {
 
     /* Try a partial resynchronization if this is a PSYNC command.
      * If it fails, we continue with usual full resynchronization, however
-     * when this happens masterTryPartialResynchronization() already
-     * replied with:
+     * when this happens masterTryPartialResynchronization() will not
+     * reply with:
```

Comment:
fixed in 72aa376

Reply:
i agree with the updated part.
but actually I think the original comment is correct. 
please take a deep look again and share your thought.